server/utils/utils.ts

```diff
@@ -499,3 +499,7 @@ export function isValidPathway(pathwayFromUrl: string): boolean {
 export function isValidStatus(status: string): boolean {
   return Object.keys(STATUS_DICTIONARY).includes(status)
 }
+
+export function isOptional(questionAndAnswer: ApiQuestionsAndAnswer): boolean {
```

Comment:
might be cleaner to return the Optional text here if it's available (so we don't have to have the if statement in njk)